build-livecd-root

```diff
@@ -49,7 +49,7 @@ isohybrid --partok --uefi fdi-bootable.iso
 mv fdi-bootable.iso $srcdir/fdi-bootable-${last_tag#release-}.iso
 
 echo "* Converting to initrd"
-livecd-iso-to-pxeboot $tmpdir/fdi.iso
+aux/livecd-iso-to-pxeboot $tmpdir/fdi.iso
```

Comment:
Oh I missed it and checked an old kernel when I was testing that. Thanks.

```
-rw-r--r-- 1 lzap lzap 255M Mar  4 09:52 fdi-image-3.1.0.tar
-rw-r--r-- 1 lzap lzap 215M Apr 26 12:50 fdi-image-3.1.1.tar
```

Amended the change.